cocos/core/scene-graph/scene-globals.ts

```diff
@@ -47,25 +50,100 @@ const _qt = new Quat();
 @ccclass('cc.AmbientInfo')
 export class AmbientInfo {
     @serializable
-    protected _skyColor = new Color(51, 128, 204, 1.0);
+    protected _skyColor = new Vec4(0.2, 0.2, 0.2, 1.0);
     @serializable
     protected _skyIllum = Ambient.SKY_ILLUM;
     @serializable
-    protected _groundAlbedo = new Color(51, 51, 51, 255);
+    protected _groundAlbedo = new Vec4(0.2, 0.5, 0.8, 1.0);
+
+    @serializable
+    protected _skyColor_ldr = new Vec4(0.2, 0.2, 0.2, 1.0);
+    @serializable
+    protected _skyIllum_ldr = Ambient.SKY_ILLUM;
+    @serializable
+    protected _groundAlbedo_ldr = new Vec4(0.2, 0.5, 0.8, 1.0);
 
     protected _resource: Ambient | null = null;
 
+    get skyColor_hdr () {
+        return this._skyColor;
+    }
+
+    get groundAlbedo_hdr () {
+        return this._groundAlbedo;
+    }
+
+    get skyIllum_hdr () {
+        return this._skyIllum;
+    }
+
+    get skyColor_ldr () {
+        return this._skyColor_ldr;
+    }
+
+    get groundAlbedo_ldr () {
+        return this._groundAlbedo_ldr;
+    }
+
+    get skyIllum_ldr () {
+        return this._skyIllum_ldr;
+    }
+
+    // Normalize HDR color
+    private normalizeHdrColor (color : Vec4) {
```

Comment:
No need to put here as a private function, just put it out side as a tool function

```
function normalizeHdrColor (color : Vec4) {
    ...
}
```